media/js/src/forms/EditSelectionForm.jsx

```diff
@@ -162,6 +162,14 @@ export default class EditSelectionForm extends React.Component {
                 <Button type="submit" size="sm">
                     Save
                 </Button>
+                <div className="float-right">
```

Comment:
Can the Delete button be left, with Cancel, Save on the right? I think the natural instinct is to click the right-hand button. Jira ticket has details + image.